spec/amp-video-interface.md

```diff
@@ -44,6 +44,76 @@ For an example, visit [AMP By Example](https://ampbyexample.com/components/amp-v
 
 <a id="rotate-to-fullscreen"></a>
 
+## Docking (minimize to corner)
+
+attribute: **`dock`**
+
+**Experimental feature. Setting this attribute is not currently valid.**
```

Comment:
so I assume the strategy is no experimental flag and using validation rules to keep this experimental until ready? (if so, thumbs up from me)

Reply:
Yup!